test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/AllowListVerificationProviderTests.cs

```diff
@@ -979,6 +982,98 @@ public AllowListVerificationProviderTests(SigningTestFixture fixture)
             }
         }
 
+        [CIOnlyFact]
+        public async Task GetTrustResultAsync_PackageSignedWithCertFromRepositoryCertificateList_SuccessAsync()
+        {
+            // Arrange
+            using (var dir = TestDirectory.Create())
+            using (var repoCertificate = new X509Certificate2(_trustedRepoTestCert.Source.Cert))
+            {
+                var nupkg = new SimpleTestPackageContext("A", "1.0.0");
```

Comment:
If package Id and version are not relevant then don't pass them, the default constructor for `SimpleTestPackageContext` will default to one.